### PR TITLE
The admin Investigate shows which categories are empty

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
admin: The Admin->Investigate verb will indicate investigate
topics that have had no messages logged this round. The view notes
option is always visible.
/:cl:

Investigate logs are good. Being shy to add new categories because
we're worried about clogging the list is bad.

- Removes the INVESTIGATE_CIRCUITS category, given it's currently completely unused.

image

### CODE REVIEW

File: code/modules/admin/admin_investigate.dm [hunk 1]
```diff
@@ -1,22 +1,48 @@
+#define INVESTIGATE_NOTES "notes, memos, watchlist" // Not a real investigation category
+#define INVESTIGATE_FILE(subject) file("[GLOB.log_directory]/[subject].html")
```
Comment (Jordie0608) [inline code/modules/admin/admin_investigate.dm]: I feel these defines aren't actually helpful and just bloat. They're local to the file and cover a whole 3 lines.

Comment (coiax) [reply code/modules/admin/admin_investigate.dm]: Should I remove the defines, or just undef them at the eof?

Comment (Jordie0608) [reply code/modules/admin/admin_investigate.dm]: Just remove.

**Comment by coiax** [comment]
@Jordie0608 The unit testing is breaking, but it's not my code's fault.

**Comment by Jordie0608** [comment]
Was just a rust dependency failing, restarting the build fixed.
Also merge conflicts.

**Pull request merged**